### ISSUE TITLE
Can't use AWS creds

### ISSUE THREAD

**Issue by kjamaal** [dwardu89/aws-ssm-parameter-store#14]
Documentation states this action is intended to be used with the Environments feature. I've never used that but I'm guessing the action gets identity information from that. The environments feature only works in a public repo according to the documentation https://docs.github.com/en/actions/reference/environments#about-environments. Would you mind if I updated this action to take in AWS secrets as parameters?

**Comment by dwardu89** [comment]
@kjamaal Certainly, just raise a pull request and we can discuss the changes. Anything that adds further functionality to this will be great! 
Thanks

**Comment by kjamaal** [comment]
https://github.com/dwardu89/aws-ssm-parameter-store/pull/15

I can work on those failed checks if they need to pass before review
